test/browser_reporting.js

```diff
@@ -52,6 +52,11 @@ function reportStdoutToServer(message) {
   }
 }
 
+async function skipTest(message) {
+  skipTest.skipped = true;
+  await reportResultToServer(message, 'skipped:');
```

Comment:
Why not do `'skipped:' + message` here and avoid the extra argument to reportResultToServer?

Reply:
The colon gets escaped and then doesn't match the check in python.

Reply:
Can you explain more?  Maybe we do a better fix here.. but I'm not sure exactly what it looks like.

Reply:
Actually, a better fix might be to move the `output = unquote(output)` line up to line 2405 in test/common.py